qiita_pet/handlers/study_handlers/prep_template.py

```diff
@@ -77,3 +80,43 @@ def post(self):
         else:
             raise HTTPError(400, 'Unknown prep template action: %s' % action)
         self.write(result)
+
+
+class PrepFilesHandler(BaseHandler):
+    @authenticated
+    def get(self):
+        study_id = self.get_argument('study_id')
+        prep_file = self.get_argument('prep_file')
+        prep_type = self.get_argument('type')
+
+        # TODO: Get file types for the artifact type
+        # FILE TYPE IN POSTION 0 MUST BE DEFAULT FOR SELECTED
+        file_types = supported_filepath_types(prep_type)
+
+        selected = []
+        not_selected = []
+        _, base = get_mountpoint("uploads")[0]
+        uploaded = get_files_from_uploads_folders(study_id)
+        prep = pd.read_table(join(base, study_id, prep_file), sep='\t')
+        if 'per_prefix' in prep.columns:
```

Comment:
@squirrelo the name of the column was correct "run_prefix"
What you needed to change is the name of the variable below, `per_sample` -> `per_prefix`

Please make that change.